website/index.html

```diff
@@ -143,6 +143,13 @@
                 <li><a href="https://github.com/square/wire">Wire</a>: <code>com.squareup.retrofit:converter-wire</code></li>
                 <li><a href="http://simple.sourceforge.net/">Simple XML</a>: <code>com.squareup.retrofit:converter-simplexml</code></li>
               </ul>
+              <p>Here's an example of using the <code>GsonConverterFactory</code> class to generate an implementation of the <code>GitHubService</code> interface which uses <code>Gson</code> for its deserialization.</p>
```

Comment:
nit: I wouldn't wrap "Gson" in `<code>` because while it's both the name of the library _and_ the class name, in this case I think we are referring to it more by library name.